app/main/helpers/email.py

```diff
@@ -0,0 +1,63 @@
+from flask import url_for, current_app
+import mandrill
+from itsdangerous import URLSafeTimedSerializer
+
+from .. import main
+
+
+with open("./email_templates/forgotten_password_email.html", "r") as f:
+    forgot_password_email_body = f.read()
+
+
+def send_password_email(user_id, email_address):
+    try:
+        mandrill_client = mandrill.Mandrill(main.config['MANDRILL_API_KEY'])
+        url = generate_reset_url(user_id, email_address)
+        body = forgot_password_email_body.format(url)
+        message = {'html': body,
+                   'subject': main.config['FORGOT_PASSWORD_EMAIL_SUBJECT'],
+                   'from_email': main.config['FORGOT_PASSWORD_EMAIL_FROM'],
+                   'from_name': main.config['FORGOT_PASSWORD_EMAIL_NAME'],
+                   'to': [{'email': email_address,
+                           'name': 'Recipient Name',
+                           'type': 'to'}],
+                   'important': False,
+                   'track_opens': None,
+                   'track_clicks': None,
+                   'auto_text': True,
+                   'tags': ['password-resets'],
+                   'headers': {'Reply-To': main.config['FORGOT_PASSWORD_EMAIL_FROM']},  # noqa
+                   'recipient_metadata': [
+                       {'rcpt': email_address,
+                        'values': {'user_id': 123456}}]
```

Comment:
Good spot - this was a placeholder, I meant to replace it with user id from the API, although I'm not sure we need it all really.